### ISSUE TITLE
ext detect bug

### ISSUE THREAD

**Issue by jtpittman195** [jtpittman195/disk_recover#3]
- EXT superblock found at byte 1050112 - block 1025
- EXT superblock found at byte 8388551 - block 8191
- EXT superblock found at byte 16777159 - block 16383
- EXT superblock found at byte 16777159 - block 16383

~~~
00100630  60 26 ba 58 00 00 ff ff  53 ef 01 00 01 00 00 00  |`&.X....S.......| 
00900630  60 26 ba 58 00 00 ff ff  53 ef 00 00 01 00 00 00  |`&.X....S.......| 
01900630  60 26 ba 58 00 00 ff ff  53 ef 00 00 01 00 00 00  |`&.X....S.......| 
02900630  60 26 ba 58 00 00 ff ff  53 ef 00 00 01 00 00 00  |`&.X....S.......| 
~~~

**Comment by jtpittman195** [comment]
first detection is correct, others are not

**Comment by jtpittman195** [comment]
fixed in 3b62e81aac00542e3494cd72b94fcfb844988d32
